src/components/MotionDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import { motion } from "@/types/motion";
+import { IconInfo } from "./icons/info";
+import { useLang } from "@/lib/useLang";
+
+const MotionDisplay = ({ motion }: { motion: motion | null }) => {
+  const infoslideString = useLang("infoslide");
+
+  return (
+    <section className="lg:w-[50vw] p-5 flex flex-col items-center self-center">
+      <p className="text-2xl md:text-2xl max-w-[80vw] lg:max-w-[50vw] mb-5 text-balance">
+        &quot;{motion?.motion || ""}&quot;
+      </p>
+      {motion && motion.adinfo ? (
```

Comment:
You could just change this to another &&, no?

Reply:
Why? It seems more readable this way.